src/v3/stylis-logical-plugin/index.d.ts

```diff
@@ -0,0 +1,174 @@
+// Updated type definitions for stylis 4.3 based on:
+// Type definnitions for stylis 4.2
+// Project: https://github.com/thysultan/stylis.js
+// Definitions by: Gen Hames <https://github.com/heyheyhello>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// NOTE: Not all types were updated so inaccuracies likely remain.
```

Comment:
Is this "NOTE" from you or the DT author?

Reply:
I added this comment. I'm probably going to strip this file down to only the types we use in the plugin rather than try to fully copy the types for `stylis` since many are incorrect between 4.2 and 4.3